cloud/datasync/package.json

```diff
@@ -35,10 +35,12 @@
     "@types/proxyquire": "^1.3.27",
     "del-cli": "^1.0.0",
     "mocha": "^3.4.2",
+    "mocha-typescript": "^1.1.5",
     "nyc": "^11.0.1",
     "proxyquire": "^1.8.0",
     "source-map-support": "^0.4.15",
     "ts-node": "^3.0.4",
+    "typemoq": "^1.7.0",
```

Comment:
We will need to add this to ./templates/base and makes sure that developers will also incorporate that change.